boot/Lexer.x

```diff
@@ -13,11 +13,24 @@
 #ifdef CABAL_PARSEC_DEBUG
 {-# LANGUAGE PatternGuards #-}
 #endif
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
 module Distribution.Parsec.Lexer
   (ltest, lexToken, Token(..), LToken(..)
   ,bol_section, in_section, in_field_layout, in_field_braces
   ,mkLexState) where
 
+-- [Note: boostrapping parsec parser]
```

Comment:
I'd really love it if there was a THIS FILE IS AUTOMATICALLY GENERATED warning on checked in bootstrapped copy, ideally done with the tooling. Maybe one way to do this that obviously works is, "If this file's name is FOO, then this is an automatically generated copy."